.github/workflows/ci.yml

```diff
@@ -30,6 +30,8 @@ jobs:
 
       - name: restore build & cargo cache
         uses: Swatinem/rust-cache@v2
+        with:
+          prefix-key: "rust-cache-20231109-sqlx-"
```

Comment:
We don't need per build prefixes (or the trailing `-`) as it still appends the CI job and other distinguishing keys.

Reply:
valid, changed it